Makefile

```diff
@@ -11,7 +11,7 @@ stage:
 	docker-compose -f ./docker/docker-compose.staging.yml up --build -V --remove-orphans
 
 test-e2e:
-	docker-compose -f ./docker/docker-compose.test.yml --env-file ./docker/.env.test up --renew-anon-volumes --abort-on-container-exit --exit-code-from immich_server_test --remove-orphans
+	docker-compose -f ./docker/docker-compose.test.yml --env-file ./docker/.env.test -p immich-test-e2e up  --renew-anon-volumes --abort-on-container-exit --exit-code-from immich-server-test --remove-orphans --build
```

Comment:
Cool, didn't know about the `-p` option to set the project name so we don't have `docker` as the project name for all docker-compose "stacks" 👍

Reply:
Yeah, usually, it would take the folder where the docker-compose file stays to name the stack. You can set this as well to override that behavior